Sources/BlogModule/Apis/BlogPostApi.swift

```diff
@@ -24,11 +24,33 @@ struct BlogPostApi: FeatherApiRepresentable {
     typealias PatchObject = BlogPostPatchObject
     
     func mapList(model: Model) -> ListObject {
-        .init(id: model.id!, title: model.title, imageKey: model.imageKey, excerpt: model.excerpt)
+        .init(id: model.id!, title: model.title, imageKey: model.imageKey, excerpt: model.excerpt, updated_at: model.updatedAt, created_at: model.createdAt, deleted_at: model.deletedAt)
     }
     
     func mapGet(model: Model) -> GetObject {
-        .init(id: model.id!, title: model.title, imageKey: model.imageKey, excerpt: model.excerpt, content: model.content, categories: [], authors: [])
+        var apiCategories = [BlogCategoryApi.ListObject]()
+             for catmodel in model.categories {
+                 apiCategories.append(BlogCategoryApi.ListObject.init(id: catmodel.id!,
+                                                                      title: catmodel.title,
+                                                                      imageKey: catmodel.imageKey,
+                                                                      color: catmodel.color,
+                                                                      priority: catmodel.priority,
+                                                                      updated_at: catmodel.updatedAt,
+                                                                      created_at: catmodel.createdAt,
+                                                                      deleted_at: catmodel.deletedAt)
+                 )
+             }
+             var apiAuthors = [BlogAuthorApi.ListObject]()
+             for authmodel in model.categories {
+                 apiAuthors.append(BlogAuthorApi.ListObject.init(id: authmodel.id!,
```

Comment:
You could use the BlogAuthorApi mapList method instead of mapping the object "by hand". 😉

Reply:
I'll try that

Reply:
Indeed easier :)